Assets/Scripts/HUDManager.cs

```diff
@@ -0,0 +1,35 @@
+using System.Collections;
+using System.Collections.Generic;
+using Unity.VisualScripting;
+using UnityEngine;
+using UnityEngine.UIElements;
+
+public class HUDManager : MonoBehaviour
+{
+    public static HUDManager Instance;
+    [SerializeField]
+    private GameObject hudObject;
+    private VisualElement hud;
+    private VisualElement moneyContainer;
+    private Label moneyDisplay;
+
+    // Start is called before the first frame update
+    void Awake()
+    {
+        Instance = this;
+        hud = hudObject.GetComponent<UIDocument>().rootVisualElement;
+    }
+
+    private void OnEnable(){
```

Comment:
There is inconsistent use of curly braces `{ }`, here they are in JS standard, but in `Awake()` the braces are in C# standard